src/pages/home/home-page/home.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
Please, rename file to the `home-page` to match with the name of the folder

Reply:
Renamed